REVIEW_GUIDELINES.md

```diff
@@ -43,6 +43,7 @@ There are a few different types of pull requests that are opened against this re
         * The license must be reflected in the package.xml file of all sub-packages in the repository.
         * The source repository must be publicly accessible.
         * The source repository should contain one or more ROS packages (meaning they have a `package.xml` in the source repository). Packages that are not ROS packages can be accepted, but they are rare and require special handling in the release repository.
+        * The source repository should generally *not* be a fork of an existing ROS package. If it is a fork, then the reviewer should inquire why the fork was made, and what efforts have been made to contact the upstream. The submitter should have at least opened an issue on the upstream asking for a release, then waited for a response. If there is no response after 4 weeks, then the reviewer can allow the fork. Note that these rules are general guidelines, rather than hard-n-fast rules; each of these situations is different and may require some different handling.
```

Comment:
```suggestion
        * The source repository should generally *not* be a fork of an existing ROS package. If it is a fork, then the reviewer should inquire why the fork was made, and what efforts have been made to contact the upstream. The submitter should have at least opened an issue on the upstream asking for a release, then waited for a response. If there is no response after 4 weeks, then the reviewer can allow the fork. Note that these rules are general guidelines, rather than hard-and-fast rules; each of these situations is different and may require some different handling.
```